server/src/utils/healthcheck.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const port = Number(process.env.IMMICH_PORT) || 3001;
+const controller = new AbortController();
+
+const main = async () => {
+  const timeout = setTimeout(() => controller.abort(), 2000);
+  try {
+    const response = await fetch(`http://localhost:${port}/api/server-info/ping`, {
+      signal: controller.signal,
+    });
+
+    if (response.ok) {
+      const body = await response.json();
+      if (body.res === 'pong') {
+        console.log('Server is running');
```

Comment:
Will this write to the docker logs every time the HC is run (default 30 seconds)? Is that desired?

Reply:
Doing a watch on the `immich_server` container for 2 minutes and nothing showed up (healthchecks should by default happen every 30 seconds). There is no other logs than the container's stdout right?

Reply:
Hm, interesting. I guess that should be fine then

Reply:
Imo just remove it anyway, if it doesn't get written anywhere then there's no point having it

Reply:
removed 2 of the 3 console calls. I left the last one in because there is a very minor usefulness to them. If healthchecks are failing when they shouldn't be you could exec in and run the file with node. Then look at the output for debugging.

Let me know what you think. I'm good either way.

Reply:
Looks good to me, thanks for making all these changes!

Reply:
It's not really important here, but for future reference, healthcheck logs are captured separately by docker and listed in the container status next to healthcheck status changes.